core/src/main/java/io/parsingdata/metal/data/ParseGraph.java

```diff
@@ -119,11 +119,11 @@ ParseGraph closeBranch() {
 
     @Override
     public String toString() {
-        if (this == EMPTY) { return "graph(EMPTY)"; }
+        if (this == EMPTY) { return "pg(EMPTY)"; }
```

Comment:
I personally always explicitly think about *parse*graph, *parse*value, etc. so I think the _p_ makes it stand out, but that may be personal.